src/Polymake.jl

```diff
@@ -42,7 +42,7 @@ function Base.showerror(io::IO, ex::PolymakeError)
 end
 
 function check_jlcxx_version(version)
-    current_jlcxx = VersionNumber(unsafe_string(ccall(:cxxwrap_version_string, Cstring, ())))
+    global current_jlcxx = VersionNumber(unsafe_string(ccall(:cxxwrap_version_string, Cstring, ())))
```

Comment:
```suggestion
    current_jlcxx = VersionNumber(unsafe_string(ccall(:cxxwrap_version_string, Cstring, ())))
```

Not necessary (see below), and also bad style. If we would need something like this, the best would be something like

```julia
const current_jlcxx = Ref(VersionNumber("0.0.0"))
function check_jlcxx_version(version)
     current_jlcxx[] = VersionNumber(unsafe_string(ccall(:cxxwrap_version_string, Cstring, ())))	

end
```